### PR TITLE
Add main

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please merge

**Comment by pandalion** [comment]
👍

**Comment by henrahmagix** [comment]
👍

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "grunt-incuna-plugins",
+  "main": "index.js",
```
Comment (henrahmagix) [inline package.json:3]: This is the default value for `main`, I think, but good to be explicit

**Pull request merged**